docs/components/label.md

```diff
@@ -0,0 +1,42 @@
+# Label
+
+**KLabel** provides a wrapper around general `label` tags
+
+## Standard Input
+
+<br />
+
+<KLabel>Input Title</KLabel>
+
+```vue
+<KLabel>Input Title</KLabel>
+```
+
+## With Help
+
+<br />
+
+<KLabel help="This is an example">Input Title</KLabel>
+
+```vue
+<KLabel help="This is an example">Input Title</KLabel>
+```
+
+<br />
+
+<KLabel help="This is a really long tooltip. Hopefully we won't have anything this long but we might. I wonder how it handles long inputs">Long Input Title</KLabel>
+
+```vue
+<KLabel
+  help="This is a really long tooltip. Hopefully we won't have anything this long but we might. I wonder how it handles long inputs"
+>
+  Long Input Title
+</KLabel>
+```
+
+## Sample with an input + tooltip
+
+<br />
+
+<KLabel tooltip="A service is an API that you want to offer">Service Name</KLabel>
```

Comment:
Not seeing any handling in `KLabel` for a `tooltip` prop. Please also include a `vue` code section so the user can see the code.

![image](https://user-images.githubusercontent.com/67973710/120498673-678c8700-c38d-11eb-83d8-5e0c0ac8d32a.png)

Reply:
Good catch, the prop was renamed to `help`